rest-api/src/main/java/gov/cms/qpp/conversion/api/controllers/v1/CpcFileControllerV1.java

```diff
@@ -65,13 +62,13 @@
 			throws IOException {
 		API_LOG.info("CPC+ file request received");
 
-		InputStream inStream = cpcFileService.getFileById(fileId);
+		String content = cpcFileService.getFileById(fileId);
 
 		API_LOG.info("CPC+ file request succeeded");
 
 		HttpHeaders httpHeaders = new HttpHeaders();
 		httpHeaders.setContentType(MediaType.APPLICATION_XML);
 
-		return new ResponseEntity<>(IOUtils.toString(inStream, Charset.defaultCharset()), httpHeaders, HttpStatus.OK);
+		return new ResponseEntity<>(content, httpHeaders, HttpStatus.OK);
```

Comment:
I still don't like the idea of serializing up to a 40 MB (and possibly even higher) file in memory.  I would look into `InputStreamResource` and return that inside the `ResponseEntity`.